.gitignore

```diff
@@ -60,3 +60,6 @@ default.profraw
 **errorutil_analyze_errors.json
 **errorutil_analyze_summary.json
 **errorutil_errors_export.json
+
+# Temporary test files
+/mesheryctl/pkg/utils/fixtures/.meshery/
```

Comment:
don't add the path to gitignore instead don't include the files manually while making the commit.